backend/packages/Upgrade/src/api/models/FeatureFlagSegmentExclusion.ts

```diff
@@ -9,10 +9,16 @@ export class FeatureFlagSegmentExclusion extends BaseModel {
   @JoinColumn()
   public segment: Segment;
 
-  @OneToOne(() => FeatureFlag, (featureFlag) => featureFlag.featureFlagSegmentExclusion, {
+  @ManyToOne(() => FeatureFlag, (featureFlag) => featureFlag.featureFlagSegmentExclusion, {
     onDelete: 'CASCADE',
     primary: true,
   })
   @JoinColumn()
   public featureFlag: FeatureFlag;
+
+  @Column()
+  public enabled: boolean;
+
+  @Column()
+  public listType: string;
```

Comment:
We can use enum in the database but if we don't want that we should use string literals in the type.